system/include/compat/emmintrin.h

```diff
@@ -1007,7 +1007,7 @@ _mm_cvtsd_si64(__m128d __a)
 {
   // TODO: optimize
   if (isnan(__a[0]) || isinf(__a[0])) return 0x8000000000000000LL;
-  long long x = llrint(__a[0]);
+  long long x = llrintf(__a[0]);
```

Comment:
But `__m128d` is f64x2, so operating on doubles should be more correct.

Reply:
Do is the `fabsf` on the line below wrong in this case then?

Reply:
Updated the line below instead.